jaguar-gpu/jaguar-gpu.c

```diff
@@ -1,210 +1,282 @@
 /*
  * Disassembler for the Jaguar GPU/DSP
- * Copyright (C) 2018 - Sebastien Alaiwan
+ * SPDX-FileCopyrightText: 2018 Sebastien Alaiwan
```

Comment:
Could you please use the `//` style comments? Also missing the license, also better if they are before the the rest of the comments, like in Rizin itself.

Reply:
What should I add as the license? 
` LGPL-3.0-only`?